src/targetValidationUtils.js

```diff
@@ -18,6 +19,11 @@ export default function getValidationProperties(clientX, clientY, control, paper
   }
 
   if (isDraggingBoundaryEvent(control.bpmnType)) {
+    if (isBoundaryErrorEvent(control.type)) {
```

Comment:
This nested if statement is pretty janky. I'm sure there's a better way of expressing this.

Reply:
This was discussed in person and I will leave it for now. Splitting it out will mean modifying the `isBoundaryErrorEvent` function and introducing some extra complexity that may not be useful at this moment.

Since the error boundary event validation is a special case, I think this is fine for now, though it can be refactored when more boundary event validation is required.